src/main/emme/toolbox/assignment/build_transit_scenario.py

```diff
@@ -247,10 +247,10 @@ def __call__(self, period, base_scenario, transit_emmebank, scenario_id, scenari
 
             bus_fares = {}
             for mode_id, fares in mode_groups["bus"]:
-                for fare, count in fares.items():
+                for fare, count in list(fares.items()):
```

Comment:
@inrokevin do we have to cast to a list explicitly here (in new EMME version)? @JiaXu1024 said she was getting an error if she does not.

Reply:
I recalled that I got an error when I tested in EMME 4.5, but let me revert the change and test in EMME OpenPaths to confirm if it's needed.

Reply:
My test finished with no problem by removing list(). Let me update the other files. Thanks @bhargavasana